### PR TITLE
Adding project report criteria

### PR DESCRIPTION
**Pull request by maoo**
I've worked a bit on the quarterly report item, adding some criteria to evaluate project's health, so that we can build a report using the code that @robmoffat worked on a while ago - https://github.com/robmoffat/landscape-scanning

The first list of checks can probably be automated with not a lot of code; regarding the qualitative checks, it would be too time consuming to check all projects one by one, so we can keep it optional for now, and do it only for strategic projects.

We're planning to start coding next week.

### CODE REVIEW

File: responsibilities.md [hunk 1]
```diff
@@ -45,3 +45,18 @@ The processes needed to support the above responsibilities are still under devel
...
+
+### Project reporting
+
+In order to measure project health, the following metrics should be collected:
+- Number of code changes (commits) and proposals (Pull Requests) in the last quarter made by humans (no bots); if the sum of changes and proposals is lower than 5 (to be confirmed), then the project can be considered `inactive`.
+- Average response time from the project team to issues, pull requests; if the average response time (in the last quarter) is higher than 5 days (to be confirmed), then the project can be considered `unresponsive`.
+- CII Best Practices badge; if the project didn't apply for a badge yet, or the score is lower than 70%, then the project can be considered `not aligned with best practices`.
+- Documentation; if the project doesn't have a documentation website and the README.md length is lower than 400 words, then the project can be considered `not properly documented`.
+- SBOM report publicly; if the project has an SBOM available in SPDX format, which demonstrates that no CVEs and license violations are found, then the project can be considered `fully compliant`.
+
```
Comment (psmulovics) [inline responsibilities.md:57]: Let's add OpenSSF scorecard to the list as well - a visualizer is like https://securityscorecards.dev/viewer/?uri=github.com/finos/fdc3-dotnet

File: responsibilities.md [hunk 2]
```diff
@@ -45,3 +45,18 @@ The processes needed to support the above responsibilities are still under devel
 ### Board reporting
 
  - It is the responsibility of the Chair to provide regular and timely reports for the Governing Board
+
+### Project reporting
```
Comment (psmulovics) [inline responsibilities.md:49]: We would start with projects, and we would expand it to SIGs and Standards in a later batch

File: responsibilities.md [hunk 3]
```diff
@@ -45,3 +45,18 @@ The processes needed to support the above responsibilities are still under devel
...
+
+In order to measure project health, the following metrics should be collected:
+- Number of code changes (commits) and proposals (Pull Requests) in the last quarter made by humans (no bots); if the sum of changes and proposals is lower than 5 (to be confirmed), then the project can be considered `inactive`.
+- Average response time from the project team to issues, pull requests; if the average response time (in the last quarter) is higher than 5 days (to be confirmed), then the project can be considered `unresponsive`.
+- CII Best Practices badge; if the project didn't apply for a badge yet, or the score is lower than 70%, then the project can be considered `not aligned with best practices`.
+- Documentation; if the project doesn't have a documentation website and the README.md length is lower than 400 words, then the project can be considered `not properly documented`.
+- SBOM report publicly; if the project has an SBOM available in SPDX format, which demonstrates that no CVEs and license violations are found, then the project can be considered `fully compliant`.
```
Comment (psmulovics) [inline responsibilities.md:56]: We don't accept cyclone or other formats?

File: responsibilities.md (hunk 1 above)
Comment (maoo) [reply responsibilities.md:61]: Added!

File: responsibilities.md (hunk 3 above)
Comment (maoo) [reply responsibilities.md]: yes, added!

File: responsibilities.md (hunk 2 above)
Comment (maoo) [reply responsibilities.md]: Agreed. Added.

**Review by psmulovics — APPROVED** [review]

**Pull request merged**